src/client/app/pages/DetailsPage/DetailsPage.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import styles from './DetailsPage.less'
+import Page from '../../components/Page'
+import MovieList from '../../components/MovieList'
+import MovieDetails from '../../components/MovieDetails'
+
+const movies = [
```

Comment:
It's better to move such constants (even mocks) into a separate config-like file and then import it directly where it will be required